pkg/controllers/cspi-controller/handler.go

```diff
@@ -526,6 +527,30 @@ func (c *CStorPoolInstanceController) reconcileVersion(cspi *cstor.CStorPoolInst
 	return cspi, nil
 }
 
+// markCSPIStatusToOffline will fetch all the CSPI resources present
+// in etcd and mark it's own CSPI.Status to Offline
+func (c *CStorPoolInstanceController) markCSPIStatusToOffline() {
+	cspiList, err := c.clientset.CstorV1().CStorPoolInstances("").List(metav1.ListOptions{})
+	if err != nil {
+		klog.Errorf("Failed to fetch CSPI list error: %v", err)
+	}
+	// Fetch CSPI UUID of this pool-manager
+	cspiID := os.Getenv(OpenEBSIOCSPIID)
```

Comment:
`OpenEBSIOCSPIID`, can this env made a little more readable. ?

Reply:
Basically it is representing `openebs.io/cspi-id`. Any suggestions?